Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -154,7 +155,7 @@ preset of the settings module is available. It can be found in
 .. include:: /Images/AutomaticScreenshots/AdminTools/PasswordHashingSettings.rst.txt
 
 The image shows settings for an instance that runs with frontend and backend users having their passwords
-stored as Argon2i hashes in the database. The other listed algorithms are deemed less secure, they however
+stored as Argon2i hashes in the database. The other listed algorithms are deemed less secure. They however
```

Comment:
"less secure" -> this does not fit to argon2id.

Reply:
How would you rephrase this?

Reply:
Maybe something like this: "You should use one of the Argon2 algorithms, as the other listed algorithms ..."